journey-map/public/render.mjs

```diff
@@ -31,6 +31,9 @@ export const getOptions = (journeyMaps) => {
         });
     });
 
+    disabledOptions.sort();
+    featureFlagOptions.sort();
```

Comment:
Just curious, why do we need to sort these?

Reply:
We don't need to, but it makes them more consistent (otherwise it's just whatever order they happen to be discovered in the YAML)